Task7/Task7/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task7
+{
+    //Каждое из чисел m и n нечетное. 
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите два числа");
```

Comment:
В сообщениях лучше быть точнее в формулировках
```suggestion
            Console.WriteLine("Введите два целых числа");
```